packages/grub-customizer-git/grub-customizer-git.pacscript

```diff
@@ -0,0 +1,31 @@
+name="grub-customizer-git"
+pkgname="grub-customizer"
+url="https://git.launchpad.net/${pkgname}"
+build_depends="cmake g++ libgtkmm-3.0-dev gettext libssl-dev libarchive-dev hwinfo"
+breaks="${pkgname}-deb"
+description="Grub Customizer is a graphical interface to configure the GRUB2/BURG settings and menuentries"
+maintainer="Obsidian <emil@aarestrup.nu>"
```

Comment:
```suggestion
maintainer="ObsidianPresidium <emil@aarestrup.nu>"
```

Consider using your GitHub username for automatic issue assignments.

Reply:
I use the name Obsidian in all my other pacscripts, I'll have to update those as well sometime I assume?

Reply:
Yeah